scripts/start

```diff
@@ -54,5 +80,5 @@ function onListening() {
   let bind = typeof addr === "string"
     ? "pipe " + addr
     : "port " + addr.port;
-  console.log("Listening on " + bind);
+  console.listen(" Listening on " + bind);
```

Comment:
The extra whitespace here is not required:
```suggestion
  console.listen("Listening on " + bind);
```
Since you're already appending a whitespace after the emoji in line 11.